src/client/java/teammates/client/scripts/scalabilitytests/InstructorCourseEnrollPageScalabilityTest.java

```diff
@@ -0,0 +1,56 @@
+package teammates.client.scripts.scalabilitytests;
+
+import java.io.IOException;
+
+import org.testng.annotations.Test;
+
+import teammates.client.scripts.util.Stopwatch;
+import teammates.common.util.AppUrl;
+import teammates.common.util.Const;
+import teammates.common.util.Logger;
+import teammates.test.cases.browsertests.BaseUiTestCase;
+import teammates.test.pageobjects.InstructorCourseEnrollPage;
+
+/**
+ * SUT: {@link Const.ActionURIs#INSTRUCTOR_COURSE_ENROLL_PAGE}.
+ *
+ * <p>Run InstructorCourseEnrollPageScaleTestDataGenerator.java before running the tests.
+ */
+public class InstructorCourseEnrollPageScalabilityTest extends BaseUiTestCase {
+
+    private static final Logger log = Logger.getLogger();
+
+    @Override
+    protected void prepareTestData() {
+        testData = loadDataBundle("/InstructorCourseEnrollPageScTest.json");
```

Comment:
I just realized, after pulling the latest `master`, that you're placing this JSON file in the folder for regular test data? Also, should it not be `.gitignore`-d?